### PR TITLE
test(e2e): change default for e2e testing to off

### PR DESCRIPTION
**Pull request by chriswldenyer**
Change the default to not run e2e tests.
This is to allow the nightly tests to generate develop
images for use by the new nightly-stable pipeline.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -189,7 +189,7 @@ pipeline {
   parameters {
     booleanParam(defaultValue: false, name: 'e2e_continuous')
     booleanParam(defaultValue: false, name: 'run_as_nightly')
-    booleanParam(defaultValue: true, name: 'run_e2e_test')
+    booleanParam(defaultValue: false, name: 'run_e2e_test')
```
Comment (tiagolobocastro) [inline Jenkinsfile:192]: can this variable not be overridden from the jenkins UI?

Comment (arne-rusek) [reply Jenkinsfile:192]: only when run manually IMHO (i.e. "build with parameters") - bors/scheduler runs will always use the defaults; there are also global environment variables - a bit less user friendly re UI however default could be changed without touching the code

Comment (chriswldenyer) [reply Jenkinsfile:192]: Yes it can. But we want to disable it indefinitely in the Mayastor builds. E2e tests will still be run, but in another pipeline.

**Review by arne-rusek — APPROVED** [review]
If doing what you really want - re @tiagolobocastro 's question about configurability - LGTM

**Review by GlennBullingham — APPROVED** [review]

**Comment by chriswldenyer** [comment]
bors merge

**Comment by chriswldenyer** [comment]
bors merge

**Pull request merged**